ptbcontrib/extended_keyboards/README.md

```diff
@@ -0,0 +1,44 @@
+# Extra functionality for PTB keyboards
+
+### This module contains extended keyboard classes with extra functionality for PTB keyboards.
+
+### Methods is self-descriptive.
+
+```python
```

Comment:
Here you copy-pasted the abstract base class from `keyboards.py`. However this doesn't tell a user how to *use* your contribution. Please remove this snippet and instead provide examples of how you envision this contribution to be applied in an PTB-based bot. Have a look at the other contributions for inspiration.